packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -1876,6 +1876,40 @@ account_pref:
     expires: never
     data_sensitivity:
       - interaction
+  recovery_key_submit:
+    type: event
+    description: |
+      Click on "Create" or "Change" button on account settings page to add a recovery key to the account
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-10039
+    data_reviews:
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1830504
+      - https://bugzilla.mozilla.org/show_bug.cgi?id=1844121
+    expires: never
+    data_sensitivity:
+      - interaction
+  two_step_auth_submit:
+    type: event
+    description: |
+      Click on "Add" button on account settings page for adding 2FA to account
+    send_in_pings:
+      - events
+    notification_emails:
+      - vzare@mozilla.com
+      - fxa-staff@mozilla.com
+    bugs:
+      - https://mozilla-hub.atlassian.net/browse/FXA-10039
```

Comment:
Is this the correct bug? I see that FXA-10039 is used above as well.

Reply:
These are showing up as new, but it's a weird line diff because I added the new events up top. 

Good catch though - Looks like the ticket number update might have been missed when two events were added in https://github.com/mozilla/fxa/pull/17224. I'll update in this PR.

Reply:
Cleaned up the diff so the changes are clearer.